### DISCUSSION TITLE
404.html renders differently on local and gh-pages.

### DISCUSSION THREAD

**Discussion by mdeliw** [squidfunk/mkdocs-material#2385] [Q&A] [answered]
Hello!

I compared the htmls - only slight difference. Used gh-deploy option on docker.

<img width="613" alt="Screen Shot 2021-03-03 at 10 54 50 AM" src="https://user-images.githubusercontent.com/46644001/109849739-f5ca6300-7c0e-11eb-9314-95c83ace757d.png">

github pages shows the zoomed in version of the logo.
<img width="1607" alt="Screen Shot 2021-03-03 at 10 55 12 AM" src="https://user-images.githubusercontent.com/46644001/109849740-f662f980-7c0e-11eb-939e-bde068887072.png">

**Comment by squidfunk** [comment, marked as answer]
Did you set `site_url`? This is necessary for correct asset paths on the 404 page.
